### PR TITLE
MAKE-1331: Clean up change point related work

### PR DESCRIPTION
**Pull request by zamj**

### CODE REVIEW

**Comment by zamj** [comment]
> Failing tests.

Those are the ones that are going to be fixed in [this](https://jira.mongodb.org/browse/MAKE-1290) ticket, right?

**Comment by ancostas** [comment]
Ok.

**Comment by ancostas** [comment]
@julianedwards @zamj 👀

File: model/perf.go [hunk 1]
```diff
@@ -89,9 +89,7 @@ func CreatePerformanceResult(info PerformanceResultInfo, source []ArtifactInfo,
 			ProcessedAt: createdAt,
 		},
 		populated: true,
-		Analysis: PerfAnalysis{
-			ChangePoints: []ChangePoint{},
-		},
+		Analysis:  PerfAnalysis{},
```
[the remark this answers is not in the document]
Comment (julianedwards) [reply model/perf.go:92]: Yeah if it's not a pointer, no need to explicitly set it

File: model/time_series_update.go [hunk 2]
```diff
@@ -0,0 +1,236 @@
+package model
```
Comment (julianedwards) [inline model/time_series_update.go:1]: Add documentation on public types and funcs please

File: model/time_series_update.go [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+		return nil, errors.Wrap(err, "Cannot aggregate time series ids")
+	}
+	defer cur.Close(ctx)
+	var res []PerformanceResultSeriesID
+	err = cur.All(ctx, &res)
+	if err != nil {
+		return nil, errors.Wrap(err, "Could not decode time series ids")
+	}
+	return res, nil
+}
```
Comment (julianedwards) [inline model/time_series_update.go:163]: Add space (are you using the go formatter?)

Comment (zamj) [reply model/time_series_update.go:176]: Yeah I was using `go fmt`. Weird it wasn't adding a space. Added it now.

**Pull request merged**